common/src/main/scala/com/gu/media/upload/PlutoUploadActions.scala

```diff
@@ -2,32 +2,39 @@ package com.gu.media.upload
 
 import com.gu.media.aws.{DynamoAccess, KinesisAccess, SESSettings}
 import com.gu.media.logging.Logging
+import com.gu.media.model._
 import com.gu.media.ses.Mailer
-import com.gu.media.upload.model.UploadMetadata
 import com.gu.media.{PlutoDataStore, Settings}
 
 class PlutoUploadActions(config: Settings with DynamoAccess with KinesisAccess with SESSettings) extends Logging {
   private val mailer = new Mailer(config.sesClient, config.getMandatoryString("host"))
   private val plutoStore = new PlutoDataStore(config.dynamoDB, config.manualPlutoDynamo)
 
-  def sendToPluto(metadata: UploadMetadata): Unit = {
-    val plutoData = metadata.pluto
-
-    plutoData.projectId match {
-      case Some(project) =>
-        config.sendOnKinesis(config.uploadsStreamName, plutoData.s3Key, plutoData)
-
-      case None =>
-        // TODO MRB: re-enable this once the project selector has been fixed
-//        log.info(s"Sending missing Pluto ID email user=${metadata.user} atom=${plutoData.atomId}")
-//
-//        mailer.sendPlutoIdMissingEmail(
-//          metadata.title,
-//          metadata.user,
-//          config.fromEmailAddress,
-//          config.replyToAddresses)
-//
-//        plutoStore.put(plutoData)
+  def sendToPluto(plutoIntegrationData: PlutoIntegrationData): Unit = {
+    plutoIntegrationData match {
+      case plutoData: PlutoSyncMetadata => {
+        plutoData.projectId match {
+          case Some(_) => sendKinesisMessage(plutoIntegrationData)
+          case None => {
+            // TODO MRB: re-enable this once the project selector has been fixed
```

Comment:
Is the project selector fixed in this branch already?

Reply:
sadly no, I'm doing that now in a different PR